crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -173,7 +176,10 @@ impl<T: WasmModule> WasmModuleHostActor<T> {
             tables
                 .into_iter()
                 .map(|x| (x.schema.table_name.clone(), EntityDef::Table(x))),
-            reducers.iter().map(|x| (x.name.clone(), EntityDef::Reducer(x.clone()))),
+            reducers
+                .iter()
+                .filter(|r| !(r.name.starts_with("__") && r.name.ends_with("__")))
```

Comment:
Is this not the same map that is used to resolve a reducer in `ModuleHost::call_reducer_inner`?
This would mean that `ModuleHost::call_identity_connected_disconnected` never actually calls a corresponding reducer, even if the module defines them.

Reply:
Oh I see, there's another binding